src/apps/main/home/index.js

```diff
@@ -1,20 +1,149 @@
+            <li><a data-external-link href='#Tooltip'>Tooltip</a></li>
+          </ul>
+        </div>
+        <div className='elements-content'>
+
+          <ElementHolder heading='Avatar'>
+            <div className='row'>
+              <Avatar
+                src='/images/_temp/avatar.png'
+                fullName='Chris Smith'
+                size='170'
+              />
+              <Avatar
+                fullName='Chris Smith'
+                size='170'
+              />
+            </div>
+          </ElementHolder>
+
+          <ElementHolder heading='Button'>
+            <div className='button-group'>
+              <button className='btn'>Primary</button>
+              <button className='btn btn-secondary'>Secondary</button>
+              <button className='btn btn-ghost'>Ghost</button>
+              <button className='btn btn-text'>Text</button>
+
+            </div>
+          </ElementHolder>
+
+          <ElementHolder heading='Carousel'>
+
+            <Carousel withDots>
+
+              <Image
+                srcs={[
+                  'https://source.unsplash.com/WLUHO9A_xik/200x133',
+                  'https://source.unsplash.com/WLUHO9A_xik/1200x800'
+                ]}
+              />
+
+              <Image
+                srcs={[
+                  'https://source.unsplash.com/WLUHO9A_xik/200x133',
+                  'https://source.unsplash.com/WLUHO9A_xik/1200x800'
+                ]}
+              />
+
+            </Carousel>
+
+          </ElementHolder>
+
+          <ElementHolder heading='Dropdown'>
+            <Dropdown uid='home-example' Trigger={props => <MyTrigger {...props}>Custom Trigger</MyTrigger>}>
+              <ul>
+                <li><a href='#'>Account</a></li>
+                <li><a href='#'>Settings</a></li>
+                <li><a href='#'>Log Out</a></li>
+              </ul>
+            </Dropdown>
+          </ElementHolder>
+
+          <ElementHolder heading='Form'>
+            <form>
+              <TextInput label='Text Input' />
+              <Select name='select' label='Select'>
+                <option value='volvo'>Volvo</option>
+                <option value='saab'>Saab</option>
+                <option value='mercedes'>Mercedes</option>
+                <option value='audi'>Audi</option>
+              </Select>
+              <TextArea label='Textarea' />
+
+              <div className='form-row'>
+                <Radio label='Yes' name='name' id='cool' value='cool' />
+                <Radio label='No' name='name' id='cool1' value='cool1' />
+              </div>
+
+              <Checkbox label='Checkbox' name='nameo' id='x1' value='v3' />
+            </form>
+          </ElementHolder>
+
+          <ElementHolder heading='Loading'>
+            <LoadingIndicator />
+          </ElementHolder>
+
+          <ElementHolder heading='Modal'>
+            <button className='btn' onClick={openModal}>Open Modal</button>
```

Comment:
Don't know if this is a new issue or not, but opening the Modal scrolls the page back to the top.